lib/attribute_normalizer/normalizers/downcase_normalizer.rb

```diff
@@ -0,0 +1,9 @@
+module AttributeNormalizer
+  module Normalizers
+    module DowncaseNormalizer
+      def self.normalize(value)
+        value&.downcase
```

Comment:
What do you think about adding an `allow_nil` option so the developer control the normalized type?

i.e.

```ruby
  def self.normalize(value, options = {})
    options.fetch(:allow_nil, true) ? value&.downcase : value.to_s.downcase
  end
```

Reply:
Changed!